<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #ffffff; 
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="loadButton">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #666DA3;    
    color: white;                
    border: none;
    padding: 8px 16px;
    border-radius: 6px;         
}

QPushButton:hover {
    background-color: #454973;   
}

QPushButton:pressed {
    background-color:  #454973;  
}</string>
        </property>
        <property name="text">
         <string>Загрузить</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="imageLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>37</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    background-color: #f0f0f0;     /* Цвет фона */
    color: #333333;                /* Цвет текста */
    border: 1px solid #cccccc;     /* Граница */
    border-radius: 4px;            /* Закругление углов */
    padding: 3px 18px 3px 5px;     /* Отступы */
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left: 1px solid #cccccc;
}

QComboBox::down-arrow {
    image: url(:/images/arrow_down.png); /* Опционально: своя иконка */
}</string>
        </property>
        <item>
         <property name="text">
          <string>Яркость</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Контрастность</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Грейскейл</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Насыщенность</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Температура</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Виньетка</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Резкость</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Сепия</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="valueLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider">
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
    background: #C0C0C0;          /* Цвет фона дорожки */
    height: 6px;                 /* Высота дорожки */
    border-radius: 3px;          /* Закругление дорожки */
}

QSlider::handle:horizontal {
    background: #666DA3;         /* Цвет ползунка */
    width: 16px;                /* Ширина ползунка */
    height: 16px;               /* Высота ползунка */
    margin: -5px 0;             /* Отступ ползунка */
    border-radius: 8px;         /* Круглый ползунок */
    border: 1px solid #454973;  /* Граница ползунка */
}

QSlider::sub-page:horizontal {
    background:  #808080;        /* Цвет заполненной части */
    border-radius: 3px;
}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #666DA3;    
    color: white;                
    border: none;
    padding: 8px 16px;
    border-radius: 6px;         
}

QPushButton:hover {
    background-color: #454973;   
}

QPushButton:pressed {
    background-color:  #454973;  
}</string>
        </property>
        <property name="text">
         <string>Применить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #BABECA;    
    color: white;                
    border: none;
    padding: 8px 16px;
    border-radius: 6px;         
}

QPushButton:hover {
    background-color: #979BAF;   
}

QPushButton:pressed {
    background-color:  #979BAF;  
}</string>
        </property>
        <property name="text">
         <string>Отменить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>17</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #979DC3;    
    color: white;               
    border: none;
    padding: 8px 16px;
    border-radius: 6px;          
}

QPushButton:hover {
    background-color: #7D85B5; 
}

QPushButton:pressed {
    background-color:  #7D85B5;  
}</string>
        </property>
        <property name="text">
         <string>Сохранить </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
